core/modules/replica/AddReplicaQservMgtRequest.cc

```diff
@@ -80,10 +75,13 @@ AddReplicaQservMgtRequest::AddReplicaQservMgtRequest(
         _qservRequest(nullptr) {
 }
 
-std::string AddReplicaQservMgtRequest::extendedPersistentState(SqlGeneratorPtr const& gen) const {
-    return gen->sqlPackValues(id(),
-                              databases(),
-                              chunk());
+std::map<std::string,std::string> AddReplicaQservMgtRequest::extendedPersistentState() const {
+    std::map<std::string,std::string> result;
+    result["chunk"] = chunk();
+    for (auto&& database: databases()) {
+        result["database"] = database;
+    }
```

Comment:
isn't `result["database"]` only going to contain the last database returned by `databases()`? That doesn't seem like the desired effect.

Reply:
Good catch! It's a bug. Thank you very much!